### PR TITLE
create tables in other db types

### PR DESCRIPTION
**Pull request by IsharaSandeepanie**

### CODE REVIEW

File: features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/oracle.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+CREATE TABLE IDN_REMOTE_FETCH_CONFIG (
+	ID INTEGER NOT NULL AUTO_INCREMENT,
+	TENANT_ID INTEGER NOT NULL,
+	IS_ENABLED CHAR(1) NOT NULL,
+	USER_NAME VARCHAR2(255) NOT NULL,
+	REPO_MANAGER_TYPE VARCHAR2(255) NOT NULL,
+	ACTION_LISTENER_TYPE VARCHAR2(255) NOT NULL,
+	CONFIG_DEPLOYER_TYPE VARCHAR2(255) NOT NULL,
+	ATTRIBUTES_JSON CLOB NOT NULL,
+    CONSTRAINT PK_IDN_REMOTE_FETCH_CONFIG PRIMARY KEY (ID),
```
Comment (ashensw) [inline features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/oracle.sql]: Fix formatting.

File: features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/mysql.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	CONFIG_ID INT NOT NULL,
+	FILE_PATH VARCHAR(255) NOT NULL,
+	FILE_HASH VARCHAR(255),
+	DEPLOYED_DATE TIMESTAMP,
+	DEPLOYMENT_STATUS VARCHAR(255),
+	ITEM_NAME VARCHAR(255),
+	PRIMARY KEY (ID),
+	FOREIGN KEY (CONFIG_ID) REFERENCES IDN_REMOTE_FETCH_CONFIG(ID) ON DELETE CASCADE,
+	CONSTRAINT UC_REVISIONS UNIQUE (CONFIG_ID,ITEM_NAME)
+)ENGINE INNODB;
```
Comment (ashensw) [inline features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/mysql.sql]: Add a newline.

File: features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/db2.sql [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    USER_NAME VARCHAR(255) NOT NULL,
+    REPO_MANAGER_TYPE VARCHAR(255) NOT NULL,
+    ACTION_LISTENER_TYPE VARCHAR(255) NOT NULL,
+    CONFIG_DEPLOYER_TYPE VARCHAR(255) NOT NULL,
+    ATTRIBUTES_JSON MEDIUMTEXT NOT NULL,
+    PRIMARY KEY (ID),
+    CONSTRAINT UC_CONFIG UNIQUE (REPO_MANAGER_TYPE,CONFIG_DEPLOYER_TYPE,TENANT_ID)
+);
+
+
```
Comment (ashensw) [inline features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/db2.sql]: Remove unnecessary spaces.

File: features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/oracle.sql [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    CONSTRAINT UC_CONFIG UNIQUE (REPO_MANAGER_TYPE,CONFIG_DEPLOYER_TYPE,TENANT_ID));
+
+
```
Comment (ashensw) [inline features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/oracle.sql]: Remove unnecessary spaces.

File: features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/mysql.sql (hunk 2 above)
Comment (IsharaSandeepanie) [reply features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/mysql.sql]: added a line

File: features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/oracle.sql (hunk 4 above)
Comment (IsharaSandeepanie) [reply features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/oracle.sql]: Removed spaces

File: features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/db2.sql (hunk 3 above)
Comment (IsharaSandeepanie) [reply features/org.wso2.carbon.identity.remotefetch.core.feature/resources/dbscripts/db2.sql]: Removed

**Pull request merged**